.github/workflows/api-docs-repo.yaml

```diff
@@ -1,7 +1,10 @@
 name: Publish API docs
 on:
-  workflow_dispatch:
-  # no content, allows manual triggering
+  # run when a release is published,
+  # which creates a new tag
+  push:
+    tags:
+      - v2*
```

Comment:
This disallows manual triggering - do we need to consider that case (say, for when we change the template or CSS and don't retag)? 

Suggest keeping the `workflow_dispatch` option so that we can.

Reply:
Done!